apps/meteor/tests/end-to-end/api/livechat/00-rooms.ts

```diff
@@ -76,12 +76,16 @@ describe('LIVECHAT - rooms', () => {
 	before(async () => {
 		await updateSetting('Livechat_enabled', true);
 		await updateEESetting('Livechat_Require_Contact_Verification', 'never');
+		await updateSetting('Omnichannel_enable_department_removal', true);
 		await createAgent();
 		await makeAgentAvailable();
 		visitor = await createVisitor();
 
 		room = await createLivechatRoom(visitor.token);
 	});
+	after(async () => {
+		await updateSetting('Omnichannel_enable_department_removal', true);
```

Comment:
Should it be false? Since at all other "after" block it was set to false.

Reply:
Thx!